### PR TITLE
Updated subscribers.md with necessary changes

### PR DESCRIPTION
**Pull request by akshitagupta15june**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Updated subscribers.md with necessary changes

- **Why was this change needed?** (You can also link to an open issue here)

This changes was needed to improve the readability of the sentences

"We recommend using the internal id your application uses for a specific user."---->"We recommend that you use the internal id that your application uses for a specific user..."

"In case the user want to use chat channel, he will need to....."------>"Users who wish to use chat channels will need to set their ..."

- **Other information**:

### CODE REVIEW

**Comment by akshitagupta15june** [comment]
@davidsoderberg please review these changes.

**Review by florian-lefebvre — APPROVED** [review]

**Comment by akshitagupta15june** [comment]
@florian-lefebvre all checks have passed

**Comment by florian-lefebvre** [comment]
I can't merge it but don't worry, it will eventually merged. Maintainers are quite busy with Hacktoberfest 👍

**Comment by akshitagupta15june** [comment]
thanks

File: docs/docs/platform/subscribers.md
Comment (p-fernandez) [inline docs/docs/platform/subscribers.md:36]: I consider the existing copy is clearer than the suggested one and it is correct grammatically. Also the suggested copy is not bringing any added value. Let's keep the original, please.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: docs/docs/platform/subscribers.md
Comment (akshitagupta15june) [reply docs/docs/platform/subscribers.md:36]: "internal id your application u....." this had issues while reading

Comment (akshitagupta15june) [reply docs/docs/platform/subscribers.md:36]: @p-fernandez even when you read this sentence "Data stored on the subscriber object that allows you to easily access it in the notification templates you create.. "this is also not correct and i changed it to "Data stored in the subscriber object that you can easily access in your notification templates

Comment (akshitagupta15june) [reply docs/docs/platform/subscribers.md:36]: if u want i will update this as it is "This is a unique identifier used by Novu to keep track of a specific subscriber. We recommend using the internal id your application uses for a specific user.
Using an identifier like email might cause issues locating a specific subscriber once they change their email address."

Comment (p-fernandez) [reply docs/docs/platform/subscribers.md:36]: > "internal id your application u....." this had issues while reading

I kindly disagree as stated in my original message.

Comment (akshitagupta15june) [reply docs/docs/platform/subscribers.md:36]: ok i will keep it as it is then

Comment (p-fernandez) [reply docs/docs/platform/subscribers.md:36]: > @p-fernandez even when you read this sentence "Data stored on the subscriber object that allows you to easily access it in the notification templates you create....." this is also not correct and i changed it to "Data stored in the subscriber object that you can easily access in your notification templates

I am sorry but this comment has nothing to do with the copy the rollback has been requested.

**Review by p-fernandez — APPROVED** [review]
🌟

**Comment by akshitagupta15june** [comment]
All check have passed @p-fernandez 
Thankyou

**Pull request merged**
